src/optype.rs

```diff
@@ -207,6 +213,12 @@ pub enum OpType {
     /// \end{array} \right] \f$
     CSXdg,
 
+    /// Controlled \ref OpType::S gate
```

Comment:
Use rust style for cross-references rather than doxygen style. (Several more examples of this below.)

Reply:
Fixed